packages/ui/src/components/Notification/Notification.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { Meta, StoryObj } from "@storybook/react"
+
+import { Notification } from "./Notification"
+
+type Story = StoryObj<typeof Notification>
+
+export default {
+  component: Notification,
+} satisfies Meta<typeof Notification>
+
+const Template = (
+  args: React.ComponentPropsWithoutRef<typeof Notification>,
+) => <Notification {...args} />
```

Comment:
You dont need template if you are just gonna wrap the same component.

You could add `content="Transaction has been submitted"` to Template, instead of repeating it in args for each story